core/src/com/softwarearchitecture/game_server/buttons/GridLayout.java

```diff
@@ -0,0 +1,139 @@
+package com.softwarearchitecture.game_server.buttons;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.softwarearchitecture.GameApp;
+import com.softwarearchitecture.math.Rectangle;
+import com.softwarearchitecture.math.Vector2;
+
+public class GridLayout {
```

Comment:
It might be a simpler way to implement the buttons and location. However this do work.

Reply:
I agree, but lets use this for now, until something else is implimented. Ill add this to my todo list.